### ISSUE TITLE
Inconsistent offsets in Apply To Mesh component

### ISSUE THREAD

**Issue by pmb1701** [RenderHeads/UnityPlugin-AVProVideo#625]
**Describe the bug**
When applying a vertical offset in the Apply To Mesh component, the sign of the offset needs to be inverted for Android to get the same behaviour as on Windows or Mac.

**Your Setup (please complete the following information):**
 - Unity version: 2020.2.1f1
 - AVPro Video version: 2.0.5 Core Edition
 - Operating system version: Android

**Comment by kahnivore** [comment]
Hey @pmb1701 

Could you perhaps try this with one of our sample videos and let us know if you get the same behaviour?

**Comment by pmb1701** [comment]
Yes, I get the same behaviour with the Demo_DisplayComponents scene when adding some value (let's say 0.5) to the Y offset field in the Apply To Mesh component of the cube. The video texture will be offset in opposite directions on a Windows vs. an Android build.

In our application we use the offset along with the scale property for implementing aspect ratio preservation, but it's an easy workaround to multiply the Y offset by -1 on Android vs other platforms since we set those properties through code. So we don't really need a fix, but I figured I'd report it anyway.
